google/compute_shared_operation.go

```diff
@@ -0,0 +1,22 @@
+package google
```

Comment:
The shared directory and the files inside it were removed when we moved to json serialising/deserialising to change struct versions.